.github/workflows/build.yml

```
@@ -76,9 +76,9 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up Docker Buildx
         if: steps.affected.outputs.api == 'true' || steps.affected.outputs.bots == 'true'
```

actionlint (1.7.11)

[error] key "if" is duplicated in element of "steps" section. previously defined at line:72,col:9

(syntax-check)